docs/configuration_hotfixes.md

```diff
@@ -0,0 +1,203 @@
+# Configuration Hotfixes
+
+This is an advanced guide for changing low level performance configuration on a cluster to hotfix an issue or test impact.
+
+Instructions for editing (add/remove/change) kernel arguments,sysfs,proc parameters and are described in this section.
+
+## Default tunings
+
+Default tunings are applied with the [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile, it is the base for creating a Tuned CR that would be detected by [cluster node tuning operator](https://github.com/openshift/cluster-node-tuning-operator)  and finally be executed by [tuned](https://github.com/redhat-performance/tuned).
+
+## Additional kernel arguments
+
+When creating a [performance profile CR](../deploy/crds/performance.openshift.io_v1alpha1_performanceprofile_cr.yaml) , a default set of kernel arguments are created from the [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile in addition to tuned generated argument and include:
+
+`nohz=on rcu_nocbs=<isolated_cores> tuned.non_isolcpus=<not_isolated_cpumask> intel_pstate=disable nosoftlockup tsc=nowatchdog intel_iommu=on iommu=pt systemd.cpu_affinity=<not_isolated_cores> isolcpus=<isolated_cores> default_hugepagesz=<DefaultHugepagesSize> hugepagesz=<hugepages_size> hugepages=<hugepages_>`
```

Comment:
this is fine but I can already see this list go out of sync in the neear future. I'd stress this is an example and should not take verbatim.

Reply:
Yep, it also depends on the tuned profile and we do not control it anymore.

Reply:
done